controller/up.go

```diff
@@ -0,0 +1,89 @@
+package controller
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/railwayapp/cli/entity"
+)
+
+var IGNORE_FILES = map[string]bool{
```

Comment:
I think this is good for now, but I still think that in the future we should use something like https://github.com/monochromegane/go-gitignore. Specifically to handle similar dep folders for other languages (or secret .env files).

Reply:
Done